components/upload/interface.tsx

```diff
@@ -52,6 +52,7 @@ export interface ShowUploadListInterface {
   showDownloadIcon?: boolean;
   removeIcon?: React.ReactNode | ((file: UploadFile) => React.ReactNode);
   downloadIcon?: React.ReactNode | ((file: UploadFile) => React.ReactNode);
+  previewIcon?: React.ReactNode | ((file: UploadFile) => React.ReactNode);
```

Comment:
This is not your commit

Reply:
No it is not. I am not sure how that is there

Reply:
I have solved it, I am meant to merge into feature not master